boards/arm/atsamr21_xpro/atsamr21_xpro.dts

```diff
@@ -6,6 +6,7 @@
 
 /dts-v1/;
 #include <atmel/samr21.dtsi>
+#include <atmel/samd2xx18.dtsi>
```

Comment:
The nomal is include `#include <atmel/samr21.dtsi>` inside `#include <atmel/samr2xx18.dtsi>`. This is vaiid for all models.

Reply:
I was looking on all series and I'll propose that we add samx2x14 ... samx2x18 to define the memory. This way it will fit for all sam0 series.

Reply:
Done that now, we're now down one file compared to the start, as it also covered the C20/C21 family.